rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
```

Comment:
After working offline with @smothiki on this, I wasn't aware that this functionality was [previously removed](https://github.com/deis/builder/commit/ec8b981052c5ef434d5370b5778c3167c173010e#diff-cf0e34247cf4169bf1b23bd6817d6e10L67), so essentially this is just adding it back in.